docroot/profiles/custom/acquia_drupal_cms_installer/acquia_drupal_cms_installer.info.yml

```diff
@@ -0,0 +1,49 @@
+name: Acquia Drupal CMS Installer
+type: profile
+
+# Change this to whatever your minimum required version of core is, depending on the
+# needs of the recipes you want to apply.
+core_version_requirement: '^11.1'
+
+# Change this to whatever you like. It's only visible to users in the installer's profile
+# selection step, which is skipped if you mark this profile as a distribution.
+description: 'Custom installer that allows for selection of recipes.'
+
+# Remove this entire `distribution` section if you'd like users to choose which profile
+# to use at the start of the install.
+distribution:
+  name: Acquia Drupal CMS Installer
+  install:
+    theme: drupal_cms_installer_theme
```

Comment:
For clarity, I'd maybe rename the theme to `acquia_drupal_cms_installer_theme`. I see you did that with the directory, but also the info files, functions therein, libraries, etc. I'd say take full control of the namespace.